espat/espat.go

```diff
@@ -44,8 +45,8 @@ type Config struct {
 type socket struct {
 	inUse    bool
 	protocol int
-	lip      net.IP
-	lport    int
+	lip      netip.Addr
+	lport    uint16
```

Comment:
could be paired as a netip.AddrPort as `laddr`

Reply:
fixed